README.md

```diff
@@ -1,35 +1,82 @@
+* To be a Git submodule residing in the root of your DAG's Git repoitory. 
+* The DAG project directory structure:
+```
+example_airflow_dags/
+├── Makefile
+├── Pipfile
+├── Pipfile.lock
+├── README.md
+├── docker
+│   ├── Makefile
+│   ├── README.md
+│   ├── bin
+│   │   └── start.sh
+│   ├── data
+│   │   └── ...
+│   ├── docker-compose.yml
+│   ├── docker-requirements.txt
+│   ├── example-variables.json
+│   └── worker
+│       └── dockerfile
+├── local.env
+├── example_dags
+│   ├── example_dag_1.py
+│   ├── example_dag_2.py
+│   └── ...
+├── tests
+│   ├── conftest.py
+│   ├── example_dag_1_test.py
+│   ├── example_dag_2_test.py
+│   └── ...
+└── variables.json
+```
+* The `local.env` file contains the line `DAG_DIR=some_name` with the name of the directory containing your DAG files.
 
 
 ## Setup
 
-In the root of you dags repository, run `git submodule add git@github.com:tulibraries/airflow-docker-dev-setup docker`, and then commit the changes to .gitmodules and the new `docker` directory that has just been created.
+Note: If you are starting from an existing DAG with this Git repository already contained in the `gitmodules` file, see [Setup from Existing DAG Git Repository](#FromGitModule) below instead.
 
-Next, in the root of your dags repository, create a file called `local.env`. Add the line `DAG_DIR=my_dag_dir` with the actual name of the directory in the root of the repo containing your dag files. If your dags do any python import statements, this directory name needs to match the top level package in those import statements.
+Add this Git repository as a Git submodule to the root of the DAG you are developing.
 
-Finally, you should ensure that you have set the following environment varibles, `TUPSFTP_PASSWORD`, `WORKER_SSH_KEY_PATH`, `TUP_ACCOUNT_NAME`, `TUP_SSH_KEY_PATH`, and `TUP_SFTP_ACCOUNT_NAME`. For example:
+```
+git submodule add git@github.com:tulibraries/airflow-docker-dev-setup docker`
+```
+
+Cmmit the changes to .gitmodules and the new `docker` directory that has just been created.
+
+```
+git add .gitsubmodules docker
+```
+
+In the root of your dags repository, create a file called `local.env`. add the line `DAG_DIR=my_dag_dir` with the actual name of the directory in the root of the repo containing your dag files. Following the example directory tree, the line would be: `DAG_DIR=example_dags` If your dags python code has mport statements, this DAG directory name must match the top level package in those import statements
+
+
+### <a id="FromGitModule"></a>Setup From Existing DAG Git Repository
+
+If you cloned your DAG from a Git repository and it already has the `airflow-docker-dev-setup` Git submodule, you will need to perform the following commands to complete the dev container installation process.  From your DAG's root directory:
+
+```
+	git submodule init
+	git submodule update
+```
+
+Then change directories to the docker dev environment `cd docker`
```

Comment:
Please use the actual name of the directory:
```suggestion
Then change directories to the docker dev environment `cd airflow-docker-dev-setup`
```

Reply:
Rewritten